broker-daemon/block-order-worker/index.js

```diff
@@ -573,7 +574,15 @@ class BlockOrderWorker extends EventEmitter {
     // reject the entire block order if an underlying order fails
     osm.once('reject', async () => {
       try {
-        await this.failBlockOrder(blockOrder.id, osm.order.error)
+        if (osm.shouldRetry()) {
+          const validation = async () => {
+            await this.relayer.adminService.healthCheck({})
```

Comment:
feels a little weird to have this in here. is there something more abstract we can use? like something on the top level of block order worker that is like `relayerIsAvailable`?